controls/datagrid/selection.md

```diff
@@ -6,25 +6,26 @@ position: 3
 slug: datagrid-selection-overview
 ---
 
-# Selection #
+# Selection
 
-**RadDataGrid** exposes a selection feature - you can select a cell or a row by tapping on it. The control provides single as well as multiple selection.
+**RadDataGrid** exposes a selection feature - users can select a cell or a row by tapping on it. To cancel the selection, they can simply tap on the cell or a row again. The control provides single and multiple selection.
 
-You can unselect by a tap on already selected cell/row.
-
-This article will show the basic properties and methods RadDataGrid provides for working with selection.
+This article explains the basic properties and methods that RadDataGrid provides for working with selection.
 
 ## Important Properties
 
-RadDataGrid supports different selection modes that can be modified by the following properties. First, you can choose the Unit that you will be allowed to select and then the Selection Mode:
+RadDataGrid supports different selection modes that you can modify through the [**SelectionUnit**](#selectionunit) and [**SelectedItems**](#selecteditems) properties. Before you can set the [**SelectionMode**](#selectionmode), you must choose which Unit can be selected.
```

Comment:
I suggest reorganizing this section for more clarity. Let's first introduce the properties, then add each property in a separate section with an example.

Reply:
Agree